enterprise/minio/1.0.0/questions.yaml

```diff
@@ -0,0 +1,318 @@
+            For Multi Node Multi Drive (MNMD), the entry will look like this:</br>
+            Example Entry - https://minio{1...3}.example.com:30000/data{1...4}</br></br>
+            Note that each host must use the same port number and the same number of storage items.</br>
+            In both cases /data{1..4} is the directories to be used for MinIO.
+            You have to add additional storage for each data entry.
+          schema:
+            type: boolean
+            default: false
+            show_subquestions_if: true
+            subquestions:
+              - variable: multi_mode
+                label: Multi Mode (SNMD or MNMD)
+                schema:
+                  type: list
+                  default: []
+                  items:
+                    - variable: item
+                      label: ""
+                      schema:
+                        type: string
+                        required: true
+
+        - variable: storage
+          label: Storage
+          schema:
+            type: list
+            default: []
+            items:
+              - variable: item
+                label: Storage Item
+                schema:
+                  type: dict
+                  attrs:
+                    - variable: type
+                      label: Type
+                      schema:
+                        type: string
+                        immutable: true
+                        required: true
+                        default: ixVolume
+                        enum:
+                          - value: hostPath
+                            description: Host Path
+                          - value: ixVolume
+                            description: ixVolume
+                    - variable: mountPath
+                      label: Mount Path
+                      description: The path inside the container to mount the storage.
+                      schema:
+                        type: path
+                        required: true
+                        immutable: true
+                        default: /data1
+                    - variable: hostPath
+                      label: Host Path
+                      description: The host path to use for storage.
+                      schema:
+                        type: hostpath
+                        required: true
+                        immutable: true
+                        default: ""
+                        show_if: [["type", "=", "hostPath"]]
+                    - variable: datasetName
+                      label: Dataset Name
+                      description: The name of the dataset to use for storage.
+                      schema:
+                        type: string
+                        show_if: [["type", "=", "ixVolume"]]
+                        required: true
+                        # immutable: true
+                        # Can we make this dynamic, so we can hide it?!
```

Comment:
With the `immutable` in place, I can't edit it in the initial installation. 
Also is there any way to make the "dataset" name dynamic as it's in a list.

Reply:
Latest nightly has solved the `immutable` issue.